File: src/main/java/com/sica/backend/dto/JustificationResponse.java
package com.sica.backend.dto;

import java.time.LocalDate;
import java.time.LocalDateTime;

public record JustificationResponse(
    String id,
    String asistenciaId,
    String estudianteId,
    String estudianteNombre,
    String estudianteFoto,
    String profesorId,
    String materiaId,
    String materiaNombre,
    LocalDate fecha,
    String archivoUrl,
    String archivoTipo,
    String descripcion,
    String estado,
    String comentarioProfesor,
    LocalDateTime creadoEn,
    LocalDateTime revisadoEn
) {}

File: src/main/kotlin/com/sica/backend/dto/JustificationResponse.kt
package com.sica.backend.dto

import java.time.LocalDate
import java.time.LocalDateTime

data class JustificationResponse(
    val id: String,
    val asistenciaId: String,
    val estudianteId: String? = null,
    val estudianteNombre: String? = null,
    val estudianteFoto: String? = null,
    val profesorId: String? = null,
    val materiaId: String,
    val materiaNombre: String,
    val fecha: LocalDate? = null,
    val archivoUrl: String,
    val archivoTipo: String,
    val descripcion: String,
    val estado: String,
    val comentarioProfesor: String? = null,
    val creadoEn: LocalDateTime,
    val revisadoEn: LocalDateTime? = null
)
